tools/c7n_oci/tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+import os
+from distutils.util import strtobool
+
+import pytest
+from pytest_terraform import tf
+
+from c7n.testing import PyTestUtils, reset_session_cache
+from oci_common import replace_ocid, replace_email, replace_namespace
+from tools.c7n_oci.tests.oci_flight_recorder import OCIFlightRecorder
+
+tf.LazyReplay.value = not strtobool(os.environ.get("C7N_FUNCTIONAL", "no"))
+tf.LazyPluginCacheDir.value = "../.tfcache"
+
+
+class CustodianOCITesting(PyTestUtils, OCIFlightRecorder):
+    """Pytest OCI Testing Fixture"""
+
+
+@pytest.fixture(scope="function")
+def test(request):
+    test_utils = CustodianOCITesting(request)
+    test_utils.addCleanup(reset_session_cache)
+    return test_utils
+
+
+@pytest.fixture(params=["WithCompartment", "WithoutCompartment"])
+def with_or_without_compartment(request, monkeypatch):
+    compartments = None
+    if request.param == "WithoutCompartment":
+        compartments = os.getenv("OCI_COMPARTMENTS")
+        monkeypatch.delenv("OCI_COMPARTMENTS", raising=False)
+    yield
+    if request.param == "WithoutCompartment":
+        monkeypatch.setenv("OCI_COMPARTMENTS", compartments)
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_xdist_auto_num_workers(config):
```

Comment:
why is this reducing the concurrency?

Reply:
if this wrt to pytest-terraform extraneous output, I just filed https://github.com/cloud-custodian/pytest-terraform/issues/65

Reply:
actually its probably this one.. https://github.com/cloud-custodian/pytest-terraform/issues/63

Reply:
pytest-terraform issues resolved in new release 0.6.6